src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+@each $day-number, $day in $start-day {
+  .calendar__days-start-day--#{$day} {
+    margin-left: calc((#{$cell-size} + 1px) * (#{$day-number} - 1));
+  }
+}
+
+@for $lenght from 28 through 31 {
+  .calendar-days--#{$lenght} {
+    @for $days from 1 through $lenght {
+      .calendar__days:nth-child(#{$days})::before {
+        display: inline;
+        width: #{$cell-size};
+        line-height: #{$cell-size};
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: calc(#{$cell-size} * 30 / 100);
+        content: "#{$days}";
+        text-align: center;
+      }
+    }
+  }
+}
+
```

Comment:
when 28/ 29/ 30 days in a month with date boxes also should disappear 
<img width="784" alt="image" src="https://user-images.githubusercontent.com/100297341/204283062-3ec2d9a8-82e8-476d-a625-6893dc989075.png">